interop_binaries/config/aggregation_job_creator.yaml

```diff
@@ -3,6 +3,7 @@
 # tests, and is not intended for production use.
 database:
   url: postgres://postgres@127.0.0.1:5432/postgres
+  check_schema_version: false
```

Comment:
Q: why do we need to set `check_schema_version` to `false` in various interop binaries? Is the schema populated in a different way there?

Reply:
It's because the interop binaries use `janus_aggregator::binary_utils::janus_main`, just like the "real" Janus components, and that function is responsible for instantiating a `Datastore` before any of the specific application's code gets to run. The interop binaries must opt out of schema version checks so that they can get to their own code (the function pointer invoked by `janus_main`), which will apply the migration scripts in the container image.